src/StrawberryShake/CodeGeneration/src/CodeGeneration.CSharp/Extensions/MethodBuilderExtensions.cs

```diff
@@ -25,6 +26,21 @@ public static MethodBuilder SetProtected(this MethodBuilder builder)
             return builder.SetAccessModifier(AccessModifier.Protected);
         }
 
+        public static MethodBuilder SetAccessModifier(
```

Comment:
I think the default can just be added to the settings than this would be more simplified.

Reply:
I *think* having a default allows for greater flexibility. There are cases where it's helpful to be able to say "I explicitly want whatever is default". But if you prefer it without I can remove it.

Reply:
Mainly, having an explicit default is typically useful in layering settings. Though that might not be applicable to strawberry shake.

Reply:
My thinking is that the explicit default is just whatever is the default on the settings object. If you leave the setting out of the settings json you just get the default. This simplifies the code and you still have an explicit default by omitting a setting. Which essentially benefits the user since he only needs to keep settings he actively sets.

Reply:
Another benefit here is when you use the tools to print the default setting the json will actually spill out what the default is instead of needing the user to look at the code.

Reply:
Fair enough. I'll change it. I'm not terribly married to the idea anyways. I just tend to like the idea of being able to "unset" things.

Reply:
Done.